quill-core/src/main/scala/io/getquill/ast/Ast.scala

```diff
@@ -104,4 +106,11 @@ case class Assignment(input: Ident, property: String, value: Ast)
 
 //************************************************************
 
+sealed trait SchemaDefinition extends Ast
```

Comment:
@fwbrasil I think I know how to parse our mini-DSL without polluting the AST. We would have to change how parsing works, from a stateless parser to a stateful parser so that we can pass a context along. If make that change, the parser will look more like the `StatefulTranformer`. The only thing that I don't how we would do is the ASTShow. WDYT?